### DISCUSSION TITLE
页面设置：页眉顶端边距、页脚底端边距，headsep，图表名行间距

### DISCUSSION THREAD

**Discussion by ANDATD20** [nuaatug/nuaathesis#90] [Q&A]
敬请指教：
新版本word毕业论文模板（2021），正文之前 页眉顶端边距、页脚底端边距均为 2.4cm；正文中 页眉顶端边距 为 1.5cm， 页脚底端边距为 1.75 cm。
[南京航空航天大学研究生学位论文撰写要求.zip](https://github.com/nuaatug/nuaathesis/files/9379694/default.zip)

1.在使用中调整 nuaathesis.cls 页面设置，发现编译出的文档中没有发生变化
![屏幕截图 2022-08-19 141040](https://user-images.githubusercontent.com/106942556/185558355-5a84e9eb-34f7-4cd6-8045-a1f6e060e2e8.png)
![屏幕截图 2022-08-19 141605](https://user-images.githubusercontent.com/106942556/185558378-7f741bba-6404-4f0b-bfc7-91b1a7bdd85e.png)

2.文章 \headsep 间距较word中小，在 nuaathesis.cls  中调整，编译文档也没有发生变化
![屏幕截图 2022-08-19 141947](https://user-images.githubusercontent.com/106942556/185558588-95fcda8b-e0e1-47f2-8470-9c89aaffdaee.png)

3.  对于图名距下文间距大的问题，使用 \setlength{\textfloatsep}{5pt}%%%%%%%%%% 与 \setlength{\intextsep}{5pt}   %%%%%%%%%%调整后可以缩小间距，但图表名的行间距貌似超过了20磅，这里不知道该如何修改

**Comment by yzwduck** [comment]
1、如果修改 nuaathesis.cls 里的 headsep 没有效果的话，大概是改错文件了。需要修改的是放在论文同目录下的 cls 文件，而不是上级文件夹里的 cls；

2、不知道 GitHub 出了什么 bug，我无法下载你提供的附件。我从其他途径获取到了文件，发现学校提供的 Word 模板 (完全) 不符合撰写要求。

| 页面设置 | 撰写要求 | 模板前文 | 模板正文 |
|:-- | --:| --:| --:|
| 上 | 3.3 cm | 3.2 cm | 2.54 cm |
| 页眉 | 2.6 cm | 2.4 cm | 1.5 cm |

并且，同样内容的页眉，应该要用统一的格式，不应该存在前文/正文两种不同格式。如果要在 LaTeX 里复现 Word 的页眉设置，按下面算式设置 geometry:
* top: Word.上；
* headheight: 15.6bp≈0.548277cm, 即 Word 里的文档网络高度；
* headsep: Word.上 - Word.页眉 - geometry.headheight；
* 不必设置 voffset, topmargin,

3、LaTeX 里的题注机制与 Word 有很大差异，很抱歉我不知道如何在 LaTeX 里实现 Word 的题注排版。
